### ISSUE TITLE
[Feature Request]  streaming SDK for OEML?

### ISSUE THREAD

**Issue by marvin-hansen** [coinapi/coinapi-sdk#109]
CoinAPI provides OEML as REST, websocket & FIX and yet only a generated REST API is available. 

Are there plans for an OEML socket streaming SDK similar to the ones listed for the data web streaming? 

Related to that, OEML has been labeled as "public beta", is there a rough ETA for release i.e. Q4 or next year?

**Comment by marvin-hansen** [comment]
Meanwhile, I wrote a basic streaming SDK for the OEML API based on the spec's to fill the gap in my project. 

https://github.com/marvin-hansen/go-ws

 I have a really hard time testing it again the OEML API docker container. 

At some point, I got a stack-trace essentially saying:

```
'CoinAPI.REST.V1.Exceptions.TooManyRequestsException'
```

Which seems to indicate the 100 req per free key was used up. 

When generating a new key, I can start the container maybe 3 - 5 times before getting the same error so whatever
the container does, it triggers the error above.  Key #3 was throwing a not-authorized error... 

```
'CoinAPI.REST.V1.Exceptions.UnauthorizedException'
```

With some more effort I made this container run, temporarily, but then it refused any incoming connection. 

I guess the only valid conclusion is to take a second look maybe next year or so.
